guides/content/developer/core/taxation.md

```diff
@@ -133,6 +129,25 @@ Finally, if the order's address is changed to being outside this tax zone, then
 
 ### Additional Examples
 
+
+#### Differing VATs for different product categories depending on the customer's zone
+
+As of January 1st, 2015, digital products sold within the EU have to have the VAT of the receiving country applied. Physical products have to have the seller's VAT applied. In order to set this up, please proceed as follows:
```

Comment:
"have to have" should be "Physical products must have the seller's VAT applied."

http://english.stackexchange.com/questions/35545/is-this-correct-id-have-to-have-had